src/views/editArticle/Index.jsx

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable react/jsx-no-bind */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Container, Alert } from 'react-bootstrap';
+import Create from '../../components/createForms/Create';
+import Preview from '../../components/previewArticle/Preview';
+import { validateArticle } from '../../utils/validator';
+import { editArticle } from '../../redux/actions/articles';
+import Header from '../../components/header/Header';
+
+const EditArticle = (props) => {
+  const { user, article } = props;
+  const [articleData, setArticleData] = useState({
+    title: article.title,
+    description: article.description,
+    body: article.body,
+    slug: article.slug,
+    image: article.images && article.images.length > 0 ? article.images[0] : 'https://via.placeholder.com/700x400',
```

Comment:
@chingsley I thought a placeholder image is being sent from the backend?

Reply:
You are right. It is. There is no need to set it from the frontend.

Reply:
Done